common/src/main/java/com/wynntils/screens/settings/lists/FeatureList.java

```diff
@@ -113,8 +104,6 @@ protected void renderBackground(PoseStack poseStack) {
                 Texture.FEATURE_LIST_BACKGROUND.height());
     }
 
-    // FIXME: This is incorrect, entries have different heights
-    //        Consider not using this in renderList
```

Comment:
How is this fixed now?

Reply:
It is not, but it does not make that big of a difference. I will fix it either way though (although not here, but in the render code where I actually use this).

Reply:
Fixed now.